settings.gradle

```diff
@@ -4,6 +4,7 @@ pluginManagement {
         gradlePluginPortal()
     }
 }
+enableFeaturePreview('STABLE_PUBLISHING')
```

Comment:
Do we need to make other changes to support this?

https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:deferred_configuration

Reply:
Nope we were never depending on any issues that would have caused.